languages/common-lisp/exercises/concept/lillys-lasagna-leftovers/.docs/instructions.md

```diff
@@ -0,0 +1,109 @@
+Lilly the Lisp Alien has definitely made a lot of Lasagna with your
+help. They want to thank you by giving you some leftovers. But first
+they have some feedback about one of the functions you helped them
+with.
+
+In this exercise you are going to write some more functions to help
+Lilly with their Lasagna cooking as well as helping them split the
+leftovers with you.
+
+You have six tasks, three about improving the existing Lasagna cooking
+functions and three about splitting the leftovers.
+
+## 1. Make `preparation-time-in-minutes` easier to use
+
+Lilly says they found using your `preparation-time-in-minutes` a
+little awkward to use since they had to count all the layers before
+using the functions. Lilly wonders if you could make a version of the
+function that takes the layers (any number of them) as arguments and
+then calculate the preparation time based upon how many layers there
+were? As a reminder, it takes 19 minutes to prepare a single layer.
+
+```lisp
+(preparation-time 'sauce 'cheese 'right-handed-macaroni 'cheese 'sauce
+                  'left-handed-macaroni 'sauce 'sauce 'cheese 'cheese)
+                  ;; => 190 (because there are 10 layers.
+```
+
+## 2. Allow changing the expected oven time
+
+While Lilly's parental-units always cooked their lasagna for 337
```

Comment:
Loving the "parental-units"! Thanks for fleshing out the whole lisp-alien thing! It definitely makes it easier for me to run with!